tools/c7n_gcp/c7n_gcp/filters/accessapproval.py

```diff
@@ -0,0 +1,53 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Access Approval Filter for GCP Project
+"""
+from c7n.filters.core import ValueFilter
+from c7n.utils import local_session, type_schema
+from c7n_gcp.resources.resourcemanager import Project
+from googleapiclient.errors import HttpError
+
+
+@Project.filter_registry.register('access-approval')
+class AccessApprovalFilter(ValueFilter):
+    """Filter Resources based on access approval configuration
+
+    .. code-block:: yaml
+
+      - name: project-access-approval
+        resource: gcp.project
+        filters:
+        - type: access-approval
+          key: enrolledServices.cloudProduct
+          value: "all"
+    """
+    schema = type_schema('access-approval', rinherit=ValueFilter.schema)
+    permissions = ('accessapproval.settings.get',)
+
+    def process(self, resources, event=None):
+        self.findings = self.get_findings()
```

Comment:
not clear why this is being stored as an instance variable.

Reply:
Removed this redundant variable.